packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -484,7 +484,7 @@ A successful revocation is denoted by an HTTP 200 OK response. Note that revokin
 | Error Id          | Details                                                                                                                                                       |
 | :---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | invalid_client    | The specified `client_id` wasn't found.                                                                                                                       |
```

Comment:
General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid